.flake8

```diff
@@ -0,0 +1,16 @@
+# See:
+#
+# https://pycodestyle.readthedocs.io/en/latest/intro.html#error-codes  (E, W)
+# https://flake8.pycqa.org/en/latest/user/error-codes.html (F)
+# https://github.com/PyCQA/flake8-bugbear
+#
+# for error codes.  And
+#
+# https://flake8.pycqa.org/en/latest/user/violations.html#selecting-violations-with-flake8
+#
+# for error classes selected below.
+
+[flake8]
+max-line-length = 80
```

Comment:
Can this be 88?

Reply:
Yes, this is a bit strange, but what is happening is that it is setting the line length limit to 80, and then ignoring that rule later on.  This is what the black docs recommend, IIRC.

Reply:
Ah, I see. Great, thanks!